<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EosKeypadForm</class>
 <widget class="QWidget" name="EosKeypadForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="keypadLayout">
     <item row="1" column="0">
      <widget class="QSquarePushButton" name="keyBlank1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSquarePushButton" name="keyDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="5" column="12">
      <widget class="QSquarePushButton" name="keySelectManual">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Select
Manual</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QSquarePushButton" name="key9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="4" column="11">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="6">
      <widget class="QSquarePushButton" name="key6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="12">
      <widget class="QSquarePushButton" name="keySelectLast">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Select
Last</string>
       </property>
      </widget>
     </item>
     <item row="9" column="4">
      <widget class="QSquarePushButton" name="keyClear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item row="9" column="9" colspan="2">
      <widget class="QPushButton" name="keyEnter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item row="7" column="10">
      <widget class="QSquarePushButton" name="keyQOnly">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>QOnly
Track</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QSquarePushButton" name="key5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="7">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="9">
      <widget class="QSquarePushButton" name="keyPercentMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>-%</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QSquarePushButton" name="keyThru">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Thru</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QSquarePushButton" name="key4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSquarePushButton" name="keyEffect">
       <property name="text">
        <string>Effect</string>
       </property>
      </widget>
     </item>
     <item row="4" column="10">
      <widget class="QSquarePushButton" name="keySneak">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>Sneak</string>
       </property>
      </widget>
     </item>
     <item row="6" column="8">
      <widget class="QSquarePushButton" name="keyOut">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Out</string>
       </property>
      </widget>
     </item>
     <item row="9" column="5">
      <widget class="QSquarePushButton" name="key0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSquarePushButton" name="keyBlank3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QSquarePushButton" name="keyPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="9" column="6">
      <widget class="QSquarePushButton" name="keyDecimal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="6" column="12">
      <widget class="QSquarePushButton" name="keySelectActive">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Select
Active</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QSquarePushButton" name="key7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSquarePushButton" name="keyPath">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item row="6" column="10">
      <widget class="QSquarePushButton" name="keyTrace">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Trace</string>
       </property>
      </widget>
     </item>
     <item row="5" column="9">
      <widget class="QSquarePushButton" name="keyPercentPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>+%</string>
       </property>
      </widget>
     </item>
     <item row="7" column="12">
      <widget class="QSquarePushButton" name="keyLast">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Last</string>
       </property>
      </widget>
     </item>
     <item row="7" column="8">
      <widget class="QSquarePushButton" name="keyFull">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Full</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="4">
      <widget class="QSquarePushButton" name="key1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QSquarePushButton" name="keyMark">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Mark</string>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QSquarePushButton" name="keyMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="7" column="6">
      <widget class="QSquarePushButton" name="key3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="5" column="8">
      <widget class="QSquarePushButton" name="keyRemDim">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Rem
Dim</string>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QSquarePushButton" name="key8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="9" column="12">
      <widget class="QSquarePushButton" name="keyNext">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item row="5" column="10">
      <widget class="QSquarePushButton" name="keyHome">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSquarePushButton" name="keyBlank2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="8">
      <widget class="QSquarePushButton" name="keyAt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>At</string>
       </property>
      </widget>
     </item>
     <item row="7" column="9">
      <widget class="QSquarePushButton" name="keyLevel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Level</string>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QSquarePushButton" name="key2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QSquarePushButton" name="keySlash">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="4">
      <widget class="QSquarePushButton" name="keyGoToCue">
       <property name="text">
        <string>Go To Cue</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QSquarePushButton" name="keyBlock">
       <property name="text">
        <string>Block</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QSquarePushButton" name="keyAssert">
       <property name="text">
        <string>Assert</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QSquarePushButton" name="keyQuery">
       <property name="text">
        <string>Query</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QSquarePushButton" name="keyCopyTo">
       <property name="text">
        <string>Copy To</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QSquarePushButton" name="keyRecallFrom">
       <property name="text">
        <string>Recall From</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QSquarePushButton" name="keyLabelNote">
       <property name="text">
        <string>Label
Note</string>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QSquarePushButton" name="keyBlank4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QSquarePushButton" name="keyAbout">
       <property name="text">
        <string>About</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QSquarePushButton" name="keyOffset">
       <property name="text">
        <string>Offset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QSquarePushButton" name="keyHigh">
       <property name="text">
        <string>High</string>
       </property>
      </widget>
     </item>
     <item row="2" column="10">
      <widget class="QSquarePushButton" name="keyFan">
       <property name="text">
        <string>Fan</string>
       </property>
      </widget>
     </item>
     <item row="2" column="12">
      <widget class="QSquarePushButton" name="keyCapture">
       <property name="text">
        <string>Capture</string>
       </property>
      </widget>
     </item>
     <item row="1" column="12">
      <widget class="QSquarePushButton" name="keyPark">
       <property name="text">
        <string>Park</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSquarePushButton" name="keyPart">
       <property name="text">
        <string>Part</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSquarePushButton" name="keyCue">
       <property name="text">
        <string>Cue</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSquarePushButton" name="keyRecord">
       <property name="text">
        <string>Record</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSquarePushButton</class>
   <extends>QPushButton</extends>
   <header location="global">qsquarepushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
